packages/flutter_svg/lib/src/loaders.dart

```diff
@@ -3,13 +3,14 @@ import 'dart:convert' show utf8;
 import 'package:flutter/foundation.dart' hide compute;
 import 'package:flutter/services.dart';
 import 'package:flutter/widgets.dart';
-import 'package:flutter_svg/src/utilities/http.dart';
```

Comment:
I read somewhere that it's not good to mix relative and absolute imports so I changed it to be more consistent.